tests/test_response.py

```diff
@@ -0,0 +1,15 @@
+from urllib import request
+
+
+class TestClass(object):
+
+    def test_response_200(self):
+        try:
+            response = request.urlopen('http://localhost:5000/')
+            response = response.getcode()
+            if response == 200:
```

Comment:
This if ... else ... could be easily eliminated.

Reply:
I know but it was supposed to be a simple test